integration/cluster_test.go

```diff
@@ -449,7 +449,17 @@ func clusterMustProgress(t *testing.T, membs []*member) {
 	kapi := client.NewKeysAPI(cc)
 	ctx, cancel := context.WithTimeout(context.Background(), requestTimeout)
 	key := fmt.Sprintf("foo%d", rand.Int())
-	resp, err := kapi.Create(ctx, "/"+key, "bar")
+	var (
+		err  error
+		resp *client.Response
+	)
+	for i := 0; i < 3; i++ {
+		resp, err = kapi.Create(ctx, "/"+key, "bar")
+		if err == nil {
+			break
+		}
+		t.Logf("WARN: create on %s error: %v. retry again ...", membs[0].URL(), err)
```

Comment:
idiomatic go messages are all lower case:
`t.Logf("failed to create key on %q (%v)", membs[0].URL(), err)`